src/core/gui/sidebar/previews/page/SidebarPreviewPages.cpp

```diff
@@ -187,13 +188,17 @@ void SidebarPreviewPages::updatePreviews() {
         this->previews.emplace_back(std::move(p));
     }
 
-    layout();
     doc->unlock();
+    layout();
```

Comment:
Why don't we need the lock anymore?

Reply:
The `layout()` command for the sidebar preview pages does not access the document `doc`.